src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,290 @@
+package duke;
+
+import duke.exceptions.EmptyCommandException;
+import duke.exceptions.UnknownCommandException;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Duke {
```

Comment:
Perhaps you can consider having a `Ui` and `Command` class to make the code more OOP?